jdaviz/configs/specviz2d/plugins/spectral_extraction/spectral_extraction.py

```diff
@@ -237,7 +252,7 @@ def marks(self):
 
         viewer2d = self.app.get_viewer('spectrum-2d-viewer')
         viewer1d = self.app.get_viewer('spectrum-viewer')
-        if not viewer2d.state.reference_data or not viewer1d.state.reference_data:
+        if not viewer2d.state.reference_data:
```

Comment:
What does the removal of the second condition do?

Reply:
previously this avoided tracebacks before the original 1d spectrum was extracted.  Now this will force that error to occur, which is now caught by the try/except and exposed in the snackbar.